Dockerfile

```diff
@@ -1,12 +1,12 @@
 # syntax=docker/dockerfile:1.3
-FROM ocaml/opam:alpine-3.15-ocaml-4.14 as build-opam2web
+FROM ocaml/opam:alpine-3.20-ocaml-4.14 as build-opam2web
 RUN sudo apk add g++ gmp-dev
-COPY . /home/opam/opam2web
+COPY --chown=opam:opam . /home/opam/opam2web
 WORKDIR /home/opam/opam2web
 ENV OCAMLRUNPARAM b
 RUN sudo mkdir -p /opt/opam2web && sudo chown opam:opam /opt/opam2web
+RUN sudo mv /usr/bin/opam-2.1 /usr/bin/opam && opam update
```

Comment:
```suggestion
RUN sudo mv /usr/bin/opam-2.2 /usr/bin/opam && opam update
```

Reply:
There are two places where we specify 2.1.  I have added a commit to update both.  However, we will need to update the base images to include an opam-2.2 binary.  I have draft PR https://github.com/ocurrent/ocaml-dockerfile/pull/215, and https://github.com/ocurrent/docker-base-images/pull/289 for this.

Reply:
as noted in https://github.com/ocurrent/ocaml-dockerfile/pull/215 this binary already exists